src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs

```diff
@@ -1083,7 +1190,7 @@ void RequireDynamicallyAccessedMembers (ref ReflectionPatternContext reflectionC
 			foreach (var uniqueValue in value.UniqueValues ()) {
 				if (uniqueValue is LeafValueWithDynamicallyAccessedMemberNode valueWithDynamicallyAccessedMember) {
 					if (!valueWithDynamicallyAccessedMember.DynamicallyAccessedMemberKinds.HasFlag (requiredMemberKinds)) {
-						reflectionContext.RecordUnrecognizedPattern ($"The {GetValueDescriptionForErrorMessage (valueWithDynamicallyAccessedMember)} " +
+						reflectionContext.RecordUnrecognizedPattern ($"{reflectionContext.SourceMethod} The {GetValueDescriptionForErrorMessage (valueWithDynamicallyAccessedMember)} " +
```

Comment:
This should be part of a different change - there are more places where we need to add something like this.

Reply:
Oops, that was not intentional.